engine/src/main/java/com/redhat/cloud/notifications/processors/drawer/DrawerProcessor.java

```diff
@@ -178,16 +178,16 @@ public String buildNotificationMessage(Event event) {
             throw new RuntimeException("Drawer notification data transformation failed", e);
         }
 
-        String message = getTemplate()
+        String message = getTemplate(event.getApplicationId(), event.getEventType().getId(), event.getOrgId())
             .data("data", dataAsMap)
-            .render();
+            .render().trim();
 
         return message;
     }
 
     @CacheResult(cacheName = "drawer-template")
-    TemplateInstance getTemplate() {
-        IntegrationTemplate integrationTemplate = templateRepository.findIntegrationTemplate(null, null, DEFAULT, "drawer")
+    TemplateInstance getTemplate(UUID applicationId, UUID eventTypeId, String orgId) {
+        IntegrationTemplate integrationTemplate = templateRepository.findIntegrationTemplate(applicationId, eventTypeId, orgId, ALL, "drawer")
             .orElseThrow(() -> new IllegalStateException("No default template defined for drawer"));
```

Comment:
The exception message could be updated. If the exception is thrown, it means we didn't find:
- an integration template for the drawer matching the event type
- a default drawer integration template

Reply:
Because the `findIntegrationTemplate` supports fallbacks between all kind of templates and the lowest level is le `default` type, this exception will be raised only if default template is missing.
It should never happends unless someone delete it from BD by mistake.